<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2007</year>
      <month>12</month>
      <day>30</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2009</year>
      <month>12</month>
      <day>6</day>
     </datetime>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;创建账户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;存钱&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;取钱&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;改日期&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="qingping1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>520</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>清屏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_account">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>创建账户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qingping2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>清屏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deposit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>存钱Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_withdraw">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>取钱Button</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_C">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>信用账户</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_S">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>储蓄账户</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_output">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>321</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_account">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>89</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>账户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>利率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fee">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string> 年费（信用账户）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_credit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>信用额（信用账户）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_DepositID">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>账户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Depositamount">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>金额</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Depositdesc">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>原因</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_WithdrawID">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>账户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Withdrawamount">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>金额</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Withdrawdesc">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>原因</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_querytime">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>时间查询账目</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Confirmdays">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NextMonth">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>167</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下个月</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditday">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>天数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_showaccount">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>167</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>显示账户信息</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_nowdate">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_chushi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_querymoney">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>金额查询账目</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
